config/settings/base.py

```diff
@@ -33,6 +33,14 @@
 DATABASES["default"]["ATOMIC_REQUESTS"] = True
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
+SECONDARY_DB_ALIAS = "default"
+
+if env.db("SECONDARY_DATABASE_URL", default=""):
+    SECONDARY_DB_ALIAS = "secondary"
```

Comment:
this should probably come from the env and be set in line 36. then it can be used in the conditional in line 38

Reply:
Raised and responded in the [above comment](https://github.com/dimagi/commcare-connect/pull/481/files#r1934098135)